### PR TITLE
reproduce failing scenario for when any language is specified

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
I've been trying to get some cjs integration tests written for my hljs plugin, but so far have been unsuccessful: https://github.com/NullVoxPopuli/highlightjs-glimmer/pull/35/files#diff-a14263a13ac66963bbbbe860d2028c6ea37cadba53cd5e8b2e7e08c2f0174713R28

I'm mostly wanting to use my custom grammar with rehype/remark, but so far, have had much struggle.
ref: https://github.com/remarkjs/remark/discussions/701

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -8,6 +8,32 @@ var js = require('highlight.js/lib/languages/javascript')
 var as = require('highlight.js/lib/languages/applescript')
 var cp = require('highlight.js/lib/languages/cpp')
 
+test('options.languages', function (t) {
+  t.equal(
+    rehype()
+      .data('settings', {fragment: true})
+      .use(highlight, {
+        languages: {
+          js: js
```
Comment (NullVoxPopuli) [inline test.js:17]: output is the same, regardless of `js` or `javascript` is used for the key

Comment (NullVoxPopuli) [reply test.js:17]: This is the `actual` of the test output:

```html
<pre><code class="hljs js language-ini">const <span class="hljs-attr">two</span> = <span class="hljs-number">2</span><span class="hljs-comment">;</span></code></pre>
```

Comment (NullVoxPopuli) [reply test.js:17]: it seems automatic language detection activated, which is incorrect here

**Comment by wooorm** [comment]
Could you check again with the latest version? It now supports the latest lowlight/highlight!

**Comment by wooorm** [comment]
This example does not specify a language: languages can be specified with `language-js`. Not with `js`. This is how highlight.js (and markdown) work

**Pull request closed without merging**